Gopkg.lock

```diff
@@ -3,18 +3,15 @@
 
 [[projects]]
```

Comment:
Why did this file need to change?

Reply:
@mcoot make sure you have the latest `dep` version

Reply:
Fixed.